### PR TITLE
Add hduoc2003 information

### PR DESCRIPTION
**Pull request by hduoc2003**

### CODE REVIEW

File: members/hduoc2003.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+- **Gender:** Male
+- **Phone number:** 0787284446
+- **Email:** hduoc2003@gmail.com
+- **Facebook:** [hduoc2003's Facebook](https://www.facebook.com/hduoc2003)
+
+## Intro
+
+### For New Members
+
+Hello everyone, this is Dược. Work hard to buy things without looking at the price.
```
Comment (VanhGer) [inline members/hduoc2003.md:21]: Working :v

File: members/hduoc2003.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+### For Our Team
+
+- I want to work with people who have the same interests and ambitions to have a sense of competition.
+- I really really admire people who have high technical skills and explain concepts and context in a simple way.
+
+### If You Want to Hear, I Could Tell You
+
+- Watching film
+- Learn something new
```
Comment (VanhGer) [inline members/hduoc2003.md:31]: ???

Comment (sonntuet1997) [reply members/hduoc2003.md:31]: ?

**Review by VanhGer — COMMENTED** [review]
You must squash all the commits into one.

**Review by haipro287 — APPROVED** [review]

**Pull request closed without merging**